AnagramSolver.Cli/Program.cs

```diff
@@ -1,10 +1,53 @@
-﻿namespace AnagramSolver.Cli
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Threading.Tasks;
+
+namespace AnagramSolver.Cli
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var builder = new ConfigurationBuilder()
+               .AddJsonFile($"appsettings.json", true, true);
+            var config = builder.Build();
+            var minLength = config["MinWordLength"];
+            int minLengthAsNumber = Int32.Parse(minLength);
+            var maxAnagrams = config["MaxNumberOfAnagrams"];
+            int maxAnagramsAsNumber = Int32.Parse(maxAnagrams);
```

Comment:
Not an error, just a tip for a future:

- Move configuration building in private method.
- Creating two variables for each configurations seems a lot (memory wise, code length wise). Two variables could be used in a case, where there are multiple calculations or methods and variable name helps understand what is going on, however `minLength` is self explanatory. For example you could combine both lines into one: 

`var minLength = config["MinWordLength"];`
`int minLengthAsNumber = Int32.Parse(minLength);`
->
`var minLength = Int32.Parse(config["MinWordLength"]);`

Reply:
done